entry/src/main/js/default/i18n/en-US.json

```diff
@@ -0,0 +1,11 @@
+{
+  "strings": {
+    "Mon": "Mon",
```

Comment:
All strings in the app handle for both chinese and english and test once in both the languages